R/BASiCS_VarThresholdSearchHVG_LVG.R

```diff
@@ -77,14 +77,26 @@ BASiCS_VarThresholdSearchVG <- function(Chain,
         100 * VarThreshold, " % ... \n")
     }
 
-    suppressMessages(
-      DetectHVG <- BASiCS_DetectVG(
-        Chain,
-        Task = Task,
-        EFDR = EFDR,
-        VarThreshold = VarThreshold
-      )
-    )
+    if(is.null(Chain@parameters$epsilon)) {
+      suppressMessages(
+        DetectHVG <- BASiCS_DetectVG(
+          Chain,
+          Task = Task,
+          EFDR = EFDR,
+          VarThreshold = VarThreshold
+        )
+      )      
+    } else {
+      suppressMessages(
+        DetectHVG <- BASiCS_DetectVG(
+          Chain,
+          Task = Task,
+          EFDR = EFDR,
+          PercentileThreshold = VarThreshold
```

Comment:
If this can be a %ile of epsilon or variance contribution fraction both uses should be documented

Reply:
I agree. I didn't do it, just because I thought we may deprecate this function. See #191 
I don't think this gets much use.

Reply:
unless you have strong objections, planning to leave as is. We can decide later whether we deprecate or not.

Reply:
I would keep it as is in release (not allow %ile usage) as VarThresholdsGrid doesn't imply that %iles are valid input, can add %ile functionality or deprecate in future

Reply:
OK - I reverted and added a stop message if epsilon is used. See 643704d